packages/core/src/test/awsService/ec2/sshKeyPair.test.ts

```diff
@@ -29,6 +30,13 @@ describe('SshKeyUtility', async function () {
             const contents = await fs.readFile(keyPath, 'utf-8')
             assert.notStrictEqual(contents.length, 0)
         })
+
+        it('uses ed25519 algorithm to generate the keys', async function () {
+            const process = new ChildProcess(`ssh-keygen`, ['-vvv', '-l', '-f', keyPath])
+            const result = await process.run()
+            // Check private key header for algorithm name
+            assert.strictEqual(result.stdout.includes('[ED25519 256]'), true)
```

Comment:
Always great to see tests exercising actual reality instead of setting up mocks!